src/Sinch/Fax/Hooks/IFaxEvent.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Sinch.Core;
+
+namespace Sinch.Fax.Hooks
+{
+    [JsonInterfaceConverter(typeof(FaxEventConverter))]
+    public interface IFaxEvent
+    {
+        /// <summary>
+        ///     The different events that can trigger a webhook
+        /// </summary>
+        [JsonPropertyName("event")]
+        public FaxEventType Event { get; }
+    }
+
+    public abstract class GenericFaxEvent
```

Comment:
Unless I'm mistaken, it seems that some properties are not declared to support an event with application/json content. I would have expected to see `file` and `fileType`